test/BaselineOutput/Common/LogisticRegression/netcoreapp31/LogisticRegression-bin-norm-CV-breast-cancer.txt

```diff
@@ -0,0 +1,700 @@
+Instance	Label	Score	Probability	Log-loss	Assigned
```

Comment:
do we need to keep h e folder name as netcoreapp31?

Reply:
I went ahead and renamed this.  I was trying to keep most of the changes to be "future proof" and not lose the context of which version a change was required for, but that's difficult here without forking more and adding a lot of fallback probing (like look for net6.0, netcoreapp3.1, etc).  I think just removing the version is the best thing to do for now.